db/migrate/20130128124852_acts_as_taggable_on_migration.rb

```diff
@@ -1,4 +1,5 @@
 class ActsAsTaggableOnMigration < ActiveRecord::Migration
```

Comment:
Yep, was thinking of doing that as well. As creating a new repository called spree-multi-blogs instead, with only the stuff we need. Tags, Categories, Disqus integration can be added later, as fresh migrations.